lib/convertkit/oauth.rb

```diff
@@ -0,0 +1,65 @@
+module ConvertKit
+  # Use this class to get an access token or refresh token using the OAuth2 flow. The client id and client secret
+  # should be obtained from ConvertKit and is required for this flow. The code should be obtained when initially connect
+  # to ConvertKit via the https://app.convertkit.com/oauth/authorize endpoint.
+  class OAuth
+    URL = 'https://app.convertkit.com/'.freeze
+    TOKEN_PATH = 'oauth/token'.freeze
+
+    def initialize(client_id, client_secret, options = {})
+      @id = client_id
+      @secret = client_secret
+      @redirect_uri = options[:redirect_uri]
+      @code = options[:code]
+      @refresh_token = options[:refresh_token]
+      @conn = ConvertKit::ConnectionHelper.get_connection(URL)
```

Comment:
can we avoid abbreviations?

Reply:
Done! https://github.com/untitledstartup/convertkit-api-ruby/pull/1/commits/6f2faf2ac9c91674155dd6a7390b1f002dfbf2a2